composer.json

```diff
@@ -220,7 +220,7 @@
         }
     },
     "prefer-stable": true,
-    "minimum-stability": "stable",
+    "minimum-stability": "dev",
```

Comment:
Is this change to `dev` needed to use the version of `awcodes/filament-tiptap-editor` you need?